src/components/TableBody.js

```diff
@@ -11,12 +14,21 @@ class TableBody extends Component {
     var lockstate = this.props.lockstate || 'unlocked';
     var total_working_hours = this.props.totaldays * 8;
     var total = 0;
+    // clear weekends from data
+    var weekday = [];
+    var i = 0;
+    Object.values(data).forEach(value => {
+      if (moment(value.event_date).isoWeekday() < 6) {
```

Comment:
Maybe being picky, but why not just `moment(value.event_date).isoWeekday() > 6` and push when that occurs? (I guess push is to continue the loop with next item?)

Reply:
you mean to skip the Object.values(data).forEach(value => { ?

Reply:
oh nevermind! got what you meant :) yes that is absolutely right

Reply:
fixed in : https://github.com/codelabsab/timereport-ui/pull/8/commits/442a9e15e23f50102792ff61305a1e3b512ca124

thnx